platform_versioned_docs/version-24.1.1/launch/advanced.mdx

```diff
@@ -59,10 +59,10 @@ Replace Nextflow process commands with command [stubs](https://www.nextflow.io/d
 
 ### Main script
 
-Nextflow will attempt to run the script named `main.nf` in the project repository by default. You can change this with the `manifest.mainScript` option or by providing the script filename to run in this field.
+Nextflow will attempt to run the script named `main.nf` in the project repository by default. A custom script filename can be configured using `manifest.mainScript` or by providing the script filename in this field.
```

Comment:
```suggestion
Nextflow will attempt to run the script named `main.nf` in the project repository by default. You can configure a custom script filename in `manifest.mainScript` or you can provide the script filename in this field.
```

Reply:
See above. 

I'm wondering if we should flip the sentences around and start with the configuration sentence? Or do the two happen independently of each other?

Reply:
I think it works well with your suggestion.